Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -166,6 +166,7 @@ class OWTestLearners(OWWidget):
 
     class Error(OWWidget.Error):
         class_required = Msg("Train data input requires a class variable")
+        too_many_classes = Msg("Too many target variables.")
```

Comment:
I like the use of "target variable" as that corresponds to the use in File and Select Columns widgets.
Could you change the messages above and below to also use that instead of "class variable"?